packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart/ForceFunctionContainerStartExecutor.ts

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
nit:  I think the convention within our projects is to name files using camelCase, and not PascalCase.

Reply:
Discussed with Jeff offline.  We're going to discuss with the larger team the convention moving forward.